tools/bump-version

```diff
@@ -35,7 +44,7 @@ const commitAndTag = async (versionName, versionCode, { deliverinoPrivateKey, ow
 
   const commitSha = commit.data.commit.sha
 
-  const releaseName = `${versionName}${versionCode ? ` - version code: ${versionCode}` : ''}`
+  const releaseName = `${developmentRelease ? '[Development release] ' : ''}${versionName}${versionCode ? `-${versionCode}` : ''}`
```

Comment:
Does this differ to the webapp? I'v seen this script in several PRs now and I'm a bit confused whether they are all the same after merging everything

Reply:
No, it does not. At least after merging all PRs.